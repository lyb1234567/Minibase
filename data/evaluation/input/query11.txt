Q(z,y,x) :- R(x,y,z)