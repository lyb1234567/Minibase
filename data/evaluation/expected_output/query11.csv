'adbs', 9, 1
'anlp', 7, 2
'ids', 2, 4
'sdp', 0, 5
'mlpr', 1, 8
'rl', 9, 8
'ppls', 9, 8